### PR TITLE
DPP-84 Energy SHR, FMDM add Triall configuration issues.

### PR DESCRIPTION
**Pull request by zoemaas**

### CODE REVIEW

File: packages/oid4vci-demo-frontend/src/configs/triall.json [hunk 1]
```diff
@@ -173,7 +173,7 @@
   "routes": [
     {
       "id": "default",
-      "vpDefinitionId": "sphereonGuest",
+      "vpDefinitionId": "triall2023",
```
Comment (sksadjad) [inline packages/oid4vci-demo-frontend/src/configs/triall.json]: not sure about this change. I've recently changed this to sphereon and the reason is that it doesn't make sense to ask for a credential that we're issuing

File: packages/oid4vci-demo-frontend/src/configs/triall.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "general": {
     "credentialName": "Triall Guest",
-    "issueCredentialType": "triall2023",
+    "issueCredentialType": "GuestCredential",
```
Comment (sksadjad) [inline packages/oid4vci-demo-frontend/src/configs/triall.json]: why we're issuing GuestCredential all of a sudden?

File: packages/oid4vci-demo-frontend/src/configs/fmdm.json [hunk 3]
```diff
@@ -172,7 +172,7 @@
   "routes": [
     {
       "id": "default",
-      "vpDefinitionId": "sphereonGuest",
+      "vpDefinitionId": "fma",
```
Comment (sksadjad) [inline packages/oid4vci-demo-frontend/src/configs/fmdm.json]: not sure about this change. I've recently changed this to sphereon and the reason is that it doesn't make sense to ask for a credential that we're issuing

File: packages/oid4vci-demo-frontend/src/configs/energy_shr.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "general": {
     "credentialName": "EnergySHR Guest",
-    "issueCredentialType": "energySHR",
+    "issueCredentialType": "GuestCredential",
```
Comment (sksadjad) [inline packages/oid4vci-demo-frontend/src/configs/energy_shr.json]: why we're issuing GuestCredential all of a sudden?

**Review by sksadjad — CHANGES_REQUESTED** [review]
not sure about the changes that you made regarding the vps and the credential type

File: packages/oid4vci-demo-frontend/src/configs/triall.json (hunk 2 above)
Comment (nklomp) [reply packages/oid4vci-demo-frontend/src/configs/triall.json]: Huh that should be the type to begin with right. This is the Type that ends up in the credential and that should by GuesCredential for multiple envs like FMA, Triall, DBC

File: packages/oid4vci-demo-frontend/src/configs/fmdm.json (hunk 3 above)
Comment (nklomp) [reply packages/oid4vci-demo-frontend/src/configs/fmdm.json]: This should match the definition id that is asking for a Sphereon Wallet Identity credential.

File: packages/oid4vci-demo-frontend/src/configs/energy_shr.json (hunk 4 above)
Comment (nklomp) [reply packages/oid4vci-demo-frontend/src/configs/energy_shr.json]: Huh that should be the type to begin with right. This is the Type that ends up in the credential and that should by GuesCredential for multiple envs like FMA, Triall, DBC

File: packages/oid4vci-demo-frontend/src/configs/triall.json (hunk 1 above)
Comment (nklomp) [reply packages/oid4vci-demo-frontend/src/configs/triall.json]: This should be the definition id that is asking for the Sphereon Wallet Identity credential

Comment (zoemaas) [reply packages/oid4vci-demo-frontend/src/configs/triall.json]: There is only a pd requesting a GuestCredential,  I created a new pd for the Sphereon Wallet Identity credential.

File: packages/oid4vci-demo-frontend/src/configs/triall.json (hunk 2 above)
Comment (zoemaas) [reply packages/oid4vci-demo-frontend/src/configs/triall.json]: This is correct, I'm just not sure why it's defined in two different places

File: packages/oid4vci-demo-frontend/src/configs/fmdm.json (hunk 3 above)
Comment (zoemaas) [reply packages/oid4vci-demo-frontend/src/configs/fmdm.json]: There is only a pd requesting a GuestCredential,  I created a new pd for Sphereon Wallet Identity credential.

File: packages/oid4vci-demo-frontend/src/configs/energy_shr.json (hunk 4 above)
Comment (zoemaas) [reply packages/oid4vci-demo-frontend/src/configs/energy_shr.json]: This is correct, I'm just not sure why it's defined in two different places

Comment (nklomp) [reply packages/oid4vci-demo-frontend/src/configs/energy_shr.json]: You mean the actionParam value? That was introduced later and probably this one was never removed, or still is being used by some code path.

File: packages/oid4vci-demo-frontend/src/configs/triall.json (hunk 2 above)
Comment (zoemaas) [reply packages/oid4vci-demo-frontend/src/configs/triall.json]: Reverted

File: packages/oid4vci-demo-frontend/src/configs/energy_shr.json (hunk 4 above)
Comment (zoemaas) [reply packages/oid4vci-demo-frontend/src/configs/energy_shr.json]: Reverted

**Review by sksadjad — APPROVED** [review]
tested it with the new configurations and it's working as expected

File: packages/oid4vci-demo-frontend/src/configs/energy_shr.json (hunk 4 above)
Comment (nklomp) [reply packages/oid4vci-demo-frontend/src/configs/energy_shr.json]: No it should not be reverted! Read please

**Review by nklomp — APPROVED** [review]

**Pull request merged**